src/pages/Swap/redirects.tsx

```diff
@@ -30,11 +26,3 @@ export function RedirectToSwap() {
     />
   )
 }
-
-export function OpenClaimAddressModalAndRedirectToSwap() {
-  const dispatch = useAppDispatch()
-  useEffect(() => {
-    dispatch(setOpenModal(ApplicationModal.ADDRESS_CLAIM))
```

Comment:
Can't we remove a lot of other code that this calls into? Does this PR render that dead code?

Reply:
no i was gonna leave all the other places that can open the claim modal for now until product decides we can remove